### DISCUSSION TITLE
Field type select - description of values

### DISCUSSION THREAD

**Discussion by octavonu** [pocketbase/pocketbase#2728] [Q&A]
Hello ,

first of all I would like to thank [ganigeorgiev](https://github.com/pocketbase/pocketbase/commits?author=ganigeorgiev) for its work. Amazing job!

I am using a lot the field type select in my models and sometime I am wondering what is the meaning of the letters or numbers which defined them. Is there any option to add descriptions to them?

Thank you

**Comment by ganigeorgiev** [comment]
If you are asking for adding optional description to the `select` option values - then at least for now I'm not planning doing it because I think it may clutter the UI.

This of course depends on your use case, but if the meaning of a select option is not clear, it may be better to just use a more descriptive value, eg. `draft,public,archived` instead of `0,1,2`.

For representing enum-like data I personally prefer using strings instead of sequential numbers or single characters, because it is more clear and easy to understand (for both admins and API consumers), and in case a new option need to be added or removed your don't have to deal with rearranging the other values.
